utensor_cgen/backend/operators.py

```diff
@@ -18,7 +21,7 @@ class OperatorFactory():
   def createOperatorSnippet(self, op_info, **kwargs):
     op_type = op_info.op_type
     if op_type not in self._operators:
-      err_msg = "unsupported op type in uTensor: {}".format(op_type)
+      err_msg = "unsupported op type in uTensor: {op.name}, {op.op_type}".format(op=op_info)
```

Comment:
This is just a thought, but we might consider creating a default "Unsupported Op" that takes on any unknown op. Rather than exiting out, we can have an option for `--ignore-unsupported` which will do it's best to continue and attempt to generate the code with a `ctx.push(new {{op.name}}(....));` bit.

Reply:
I prefer `fail fast` in python.
Because it's the whole point of dev in python.
Fail fast, iterate faster.